frontend/src/components/Content.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { Wrapper } from './GlobalStyles';
+import { API_URL } from './config';
+import Loader from './Loader';
+
+const Content = () => {
+  // Local state variables for storing data from API
+  const [mostViews, setMostViews] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [accessToken] = useState(localStorage.getItem('accessToken'));
+
+  const navigate = useNavigate();
+
+  // Send in accessToken to /top10Views endpoint, set state variable with data from API
+  useEffect(() => {
+    if (accessToken) {
+    }
+    fetch(`${API_URL}/top10Views`, {
+      method: 'get',
+      headers: {
+        Authorization: accessToken
+      }
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        setMostViews(data);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, [accessToken]);
+
+  if (loading) {
+    return <Loader />;
```

Comment:
Great that you have a loader so the user understand that it is only loading and not something wrong.